libbeat/processors/add_host_metadata/_meta/fields.yml

```diff
@@ -31,3 +31,10 @@
           type: keyword
           description: >
             OS family (e.g. redhat, debian, freebsd, windows).
+        - name: net.ip
+          description: >
+            List of IP-addresses.
```

Comment:
We can use the type `ip` here: https://www.elastic.co/guide/en/elasticsearch/reference/6.2/ip.html

Reply:
Yes, of course! I'll fix that too.